code-review/README.md

```diff
@@ -0,0 +1,119 @@
+Code Review
+===========
+
+This is forked/stolen from ^W^W inspired by
+[Thoughtbot's code review guide](https://raw.githubusercontent.com/thoughtbot/guides/master/code-review/README.md)
+
+A guide for reviewing code and having your code reviewed.
+
+Why code reviews?
+-----------------
+
+Code is reviewed before being merged into `master`. The goal of review is to ensure that changes are delivering business value to our customers - not to perfect our code.
+
+When reviewing, consider whether a change:
+
+* Delivers business value. Understand why it is being made and why it is required.
+* Does what it says it does. Try to break it.
+* Can be maintained by the most junior member of our team.
+* Meets our standards and is well written.
+
+Don't let minor technical issues block shipping.
+
+A reviewer may give +1 on a review. Small fixes need +1, complex fixes and large features +2. A branch should have at least +1 before you hit the green button.
+
+Code Review Mindset
+-------------------
+
+It's important to separate our self-worth from the code we write so that we are always open to feedback that will help us improve.
+
+When reviewing code that changes user facing components it is important that reviews play the role of a User Advocate and consider how the change will impact our users, for example:
+* Is the copy written in such a way that it would be understand by someone who has learned English as their second language?
+* Is the change consistent with the user's mental model of our system?
+
+Everyone
+--------
+
+* Accept that many programming decisions are opinions. Discuss tradeoffs
+  which you prefer, and reach a resolution quickly.
+* Ask questions; don't make demands.
+    * *What do you think about naming this `:user_id`?*
+* Ask for clarification. 
+    * *I didn't understand. Can you clarify?*
+* Avoid selective ownership of code. 
+    * Avoid these words: "mine", "not mine", "yours"
+* Avoid using terms that could be seen as referring to personal traits.
+  ("dumb", "stupid"). Assume everyone is attractive, intelligent, and
+  well-meaning.
+* Be explicit. Remember people don't always understand your intentions online.
+* Be humble. 
+    * *I'm not sure - let's look it up.*
+* Don't use hyperbole. 
+    * Avoid "always", "never", "endlessly", "nothing"
+* Don't use sarcasm.
+* Keep it real. If emoji, animated gifs, or humor aren't you, don't force
+  them. If they are, use them with aplomb.
+* Talk in person if there are too many "I didn't understand" or "Alternative
+  solution:" comments. Post a follow-up comment summarizing offline discussion.
+
+Having Your Code Reviewed
+-------------------------
+
+* Be grateful for the reviewer's suggestions.
+  * *Good call. I'll make that change.*
+* Don't take it personally. The review is of the code, not you.
```

Comment:
I humbly suggest that this is far, far easier to write than to follow. To be blunt: it feels somewhat patronizing to say to someone "here is some criticism of something you worked long and hard on - don't take this as criticism".

The problem (which is not unique to code review, but comes up in most human interactions) is that we don't make a rational decision to feel attacked or criticized: we just **feel** that way. It's not under the control of the person receiving the comments. So telling someone "don't take it personally" is rather like saying "don't feel pain when I punch you in the face".

That doesn't mean that we can't make comments: it just means we need to be aware that people's self-worth **is** invested in what they create (or it damn well should be if we want passionate people to care about what they do).